packages/shared/src/modules/users/libs/types/user-dto.type.ts

```diff
@@ -0,0 +1,6 @@
+type UserDto = {
+	email: string;
+	id: number;
```

Comment:
Should we have name here as well?

Reply:
I checked, there are also createdAt and updatedAt:
```
type UserDto = {
	createdAt: string;
	email: string;
	id: number;
	name: string;
	updatedAt: string;
};
```
Then UserDto type is identcial to other two - UserGetAllItemResponseDto, UserSignUpResponseDto.
Would it be ok to create type aliases for that case?
```
type UserGetAllItemResponseDto = UserDto;
```
```
type UserSignUpResponseDto = UserDto;
```

Reply:
Yes for now, we will remove UserGetAllItemResponseDto and will update UserSignUpResponseDto later